tests/test_crs/valid_stressng.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: stressng
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
+    port: 80
```

Hey @mkarg75, can you replace these two by:

```suggestion
    server: ES_SERVER
    port: ES_PORT
```
These names are replaced by the CI.